uint16 pid_interval
---
bool success
